docs/faq.rst

```diff
@@ -255,13 +255,16 @@ How can I use the Dockstore CLI with Python 2?
 Python 2 support ends in 2020. You can get the Python 2 requirements.txt with ``curl -o requirements.txt "https://dockstore.org/api/metadata/runner_dependencies?python_version=2"``
 but it is currently untested.
 
-How do I add other users as maintainers of a tool?
---------------------------------------------------
+How do I add other users as maintainers of a tool or worklow?
+-------------------------------------------------------------
 
 For tools registered on Quay.io and workflows registered with GitHub,
 Dockstore automatically allows users from the same Quay.io organization
```

Comment:
It was already like this, but we use the term "namespace" in our UI for tools, although Quay also refers to it as organization. Regardless, I think the answer tools and workflows needs to separated. This only answers it for workflows. I guess the process is to "refresh namespace" ?